conda_rpms/build_rpm_structure.py

```diff
@@ -80,12 +80,20 @@ def create_rpmbuild_for_tag(repo, tag_name, target):
     with open(manifest_fname, 'r') as fh:
         manifest = sorted(line.strip().split('\t') for line in fh)
 
+    spec_fname = os.path.join(repo.working_dir, 'env.spec')
+    if not os.path.exists(spec_fname):
+        raise ValueError("The tag '{}' doesn't have an environment specification.".format(tag_name))
+    with open(spec_fname, 'r') as fh:
```

Comment:
Once https://github.com/pelson/conda-rpms/pull/5 is merged, you could just reuse https://github.com/pelson/conda-rpms/pull/5/files#diff-56c430e20547a5a55d4f6736e2e205cbR28 i.e.

```python
spec = Config('env.spec')
...
    fh.write(generate.render_taggedenv(env_name, tag, pkgs, config, spec['env'])
```